lib/services/ios-project-service.ts

```diff
@@ -704,7 +704,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	private async mergeInfoPlists(projectData: IProjectData): Promise<void> {
+	private async mergeInfoPlists(buildOption: IRelease, projectData: IProjectData): Promise<void> {
```

Comment:
I'd rather we name `buildOption` argument to `buildOptions` (plural) as we could put more options in it in the future